fern/docs/pages/manual/vectordb.mdx

```diff
@@ -39,6 +39,22 @@ qdrant:
   path: local_data/private_gpt/qdrant
 ```
 
+### Milvus configuration
+
+To enable Milvus, set the `vectorstore.database` property in the `settings.yaml` file to `milvus` and install the `milvus` extra.
+
+```bash
+poetry install --extras vector-stores-milvus
+```
+
+The available configuration options are:
+| Field        | Description |
+|--------------|-------------|
+| uri     | Default is set to "./milvus_llamaindex.db" as a local file, you can also set up a more performant Milvus server on docker or k8s.|
+| token          | Pair with Milvus server on docker or k8s.|
+| collection_name         | The name of the collection, set to default "milvus_db".|
+| overwrite    | Overwrite the data in collection if it existed, set to default as True. |
+
```

Comment:
- Setting the `uri` as a local file, e.g.`./milvus_llamaindex.db`, is the most convenient method, as it automatically utilizes [Milvus Lite](https://milvus.io/docs/milvus_lite.md) to store all data in this file.
- If you have large scale of data, you can set up a more performant Milvus server on [docker or kubernetes](https://milvus.io/docs/quickstart.md). In this setup, please use the server uri, e.g.`http://localhost:19530`, as your `uri`.
- If you want to use [Zilliz Cloud](https://zilliz.com/cloud), the fully managed cloud service for Milvus, adjust the `uri` and `token`, which correspond to the [Public Endpoint and Api key](https://docs.zilliz.com/docs/on-zilliz-cloud-console#free-cluster-details) in Zilliz Cloud.

Reply:
Great contribution, thanks!

We have a couple of comments:
1. By default, all local data for PGPT is saved in `local_data/private_gpt/`. Could you change this? Right now, by default, it is creating db in root.
2. This information is really useful for setup. Can you add it to the documentation?